src/plugins/data_table/filter_view.js

```diff
@@ -190,12 +238,15 @@ var DataTableFilter = (function(options) {
         filterButtonClass: this.filterButtonClass
       }));
 
-      $(".filterMenu", this.$el).prepend(this.child("filter-menu").render().$el);
+      this.$('.toggle-filter-button').popoverMenu({
+        content: this.child("filter-menu").render().$el
```

Comment:
right here.. I think I would render the filter menu, then also render a filter-actions template and append / concatenate together